modules/core/server/views/layout.server.view.html

```diff
@@ -45,7 +45,17 @@
 
   <!--Embedding The User Object-->
   <script type="text/javascript">
-    var user = {{ user | json | safe }};
+    var user = {
+      displayName: "{{ user.displayName | escape }}",
```

Comment:
Where is this *user* object coming from? Is it getting it from the session middleware when the page is rendered?

If would be nice to have some sort of condition here that would check if the user data is populated. If it's not then set this new user object to `undefined` or `null`.